### ISSUE TITLE
Upgrade puppeteer to v0.13.0

### ISSUE THREAD

**Issue by colinrotherham** [pa11y/pa11y#358]
First of all, thanks for the v5.x Pa11y it's working great! 👍 

We're using it with https://github.com/frctl/fractal to accessibility test nearly 100 of our rapidly growing list of UI components.

We also use Puppeteer via Karma to run headless Chrome/Firefox tests and again to validate HTML. Unfortunately this can result in a triple-download of different Chromium versions.

Could you please upgrade to `puppeter@0.13.0` so we can make use of the `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` environment variable?

i.e. We'd like to already have Chromium installed via Puppeteer in Jenkins for all automated test tools to use, rapidly speeding up our builds.

Thanks

**Comment by rowanmanning** [comment]
Absolutely, we'll update puppeteer as soon as possible. If anyone gets to it before me then it should be an easy contribution to Pa11y – just base your work on the `5.x` branch

**Comment by rowanmanning** [comment]
This has been updated and released under Pa11y `5.0.0-beta.5`

**Comment by jasperswart** [comment]
@colinrotherham  

I want to use chromium of puppeteer to use to run our pa11y tests in CI.

what config ar are you using?

thanks!

**Comment by colinrotherham** [comment]
@jasperswart I presume you're hitting the sandbox errors?

We use this pa11y config, adjust for your needs:

```js
{
    allowedStandards: 'WCAG2AA',
    level: 'error',
    chromeLaunchConfig: {
        args: [
            '--no-sandbox',
            '--disable-setuid-sandbox'
        ]
    }
}
```

**Comment by jasperswart** [comment]
wow thanks!

**Comment by khrome83** [comment]
@colinrotherham -

How did you get `pa11y` running in docker?

I followed the troubleshooting page, but I have issues either starting my app (no PS is crated) when i create a `pttruser` or `pa11y` hangs if I don't.

**Comment by colinrotherham** [comment]
Sorry @khrome83 I can't help debug, but since https://github.com/pa11y/pa11y/issues/358#issuecomment-351363799 our args now have one more entry:

```js
args: [
    '--no-sandbox',
    '--disable-setuid-sandbox',
    '--disable-dev-shm-usage'
]
```

Based on notes in https://github.com/GoogleChrome/puppeteer/commit/32a7fd5f37963dbe9d22f9a2976fc1e3a0c80700
